beacons/src/android/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0'?>
+<manifest xmlns:a='http://schemas.android.com/apk/res/android' package='com.gluonhq.samples.beacons' a:versionCode='1' a:versionName='1.0'>
+    <supports-screens a:xlargeScreens="true"/>
+    <uses-sdk a:minSdkVersion="21" a:targetSdkVersion="28"/>
+    <uses-permission a:name="android.permission.INTERNET"/>
+    <uses-permission a:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission a:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission a:name="android.permission.BLUETOOTH"/>
+    <uses-permission a:name="android.permission.BLUETOOTH_ADMIN"/>
+    <uses-permission a:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <application a:label='beacons' a:icon="@mipmap/ic_launcher">
+        <activity a:name='com.gluonhq.helloandroid.MainActivity' a:configChanges="orientation|keyboardHidden">
+            <intent-filter>
+                <category a:name='android.intent.category.LAUNCHER'/>
+                <action a:name='android.intent.action.MAIN'/>
+            </intent-filter>
+        </activity>
+        <activity a:name='com.gluonhq.helloandroid.PermissionRequestActivity'/>
+    </application>
+</manifest>
```

Comment:
Do we need the manifest file because we need permission for using Bluetooth services?

Reply:
Can this be marked as resolved?

Reply:
The issue has been solved (no need to copy the assets, only the manifest), but will depend on 1.24-SNAPSHOT, or a release of 1.24

Reply:
Does this mean that most of the assets in this PR can be removed?

Reply:
All of the iOS/Android assets can be removed, yes. Only if we modify the plugin (waiting for a release...)

Reply:
Ok, assets removed, pom updated

Reply:
The manifest file can be removed now, permissions are handled by Attach.